### PR TITLE
Enable Selection of Martech, GNAV, targetTesting=off

### PR DESCRIPTION
**Pull request by JingleH**
Instead of only using ?lighthosue=on, we can now use ?martech=off&gnav=off and so on to more flexibly control what to turn on/off

Test URLs:
- Before: https://stage--express--adobecom.hlx.page/express/?gnav=off
- After: https://test-performance-flags--express--adobecom.hlx.page/express/?gnav=off

### CODE REVIEW

**Review by apganapa — DISMISSED** [review]

**Comment by JingleH** [comment]
> I would even say we just retire lighthouse=on. Missing on the backward compatibility on that doesn't seem harmful.

@qiyundai @apganapa do you think we should still keep lighthouse=on to turn off everything? Or do we want to reduce some code in our scripts.js, and rely on more fine-grained control? When we raise a PR in the future for example, we would do martech=off&gnav=off&testing=off. Less convenient but a bit more performant

**Comment by vhargrave** [comment]
> martech=off

@JingleH I'd probably still keep lighthouse=on, because I think the initial idea from the franklin folks was that we should always be getting 100 with lighthouse on, because with lighthouse on we've switched off all the elements that are not in our control.

File: express/scripts/scripts.js [hunk 1]
```diff
@@ -2284,7 +2284,11 @@ async function loadLazy(main) {
  */
 async function decoratePage() {
   window.hlx = window.hlx || {};
-  window.hlx.lighthouse = new URLSearchParams(window.location.search).get('lighthouse') === 'on';
+  const params = new URLSearchParams(window.location.search);
+  window.hlx.nomartech = params.get('martech') === 'off';
```
Comment (vhargrave) [inline express/scripts/scripts.js:2288]: it's better to have variables named as positives. So:
`window.hlx.martech = params.get('martech') !== 'off'`

- https://davej.io/2018/05/negative-naming.html
- https://medium.com/dfind-consulting/about-writing-positive-and-negative-variable-names-b42328d51e9c

Comment (JingleH) [reply express/scripts/scripts.js]: @vhargrave Great advice!
I also think that I just got @qiyundai 's idea: maybe he was suggesting that we don't use window.hlx.lighthouse but still keep the ability to deduct individual fields from it. See my latest commit.

**Review by vhargrave — APPROVED** [review]
awesome!

File: express/scripts/scripts.js (hunk 1 above)
Comment (vhargrave) [reply express/scripts/scripts.js]: ah ok, yeah that makes total sense! 🚀

**Pull request merged**